Tests/FunctionalTests/TestRig/Sources/FTRNetworkTestViewController.m

```diff
@@ -68,11 +68,13 @@ - (void)verifyReceivedData:(NSData *)data {
   }
 }
 
+#if !defined(__IPHONE_9_0) || (__IPHONE_OS_VERSION_MIN_REQUIRED < __IPHONE_9_0)
```

Comment:
so what's the way to do iboutlets on 9 and above?

Reply:
Could you clarify this comment? I think IBOutlets are fine in iOS 9, it's the contents of the method that don't work on iOS 9.

So I think the code is incorrect, and I can move the #if inside the method. The button uses NSURLConnection, which is deprecated on iOS 9. So I don't know if we should replace the contents with something else. The replacement is NSURLSession, which are tested in other methods.

Let me know if you want me to move the #if into the method (and add a comment explaining what is going on). If I'm missing the point or you have other suggestions, let me know.

Reply:
I misread your cl description. If the problem is that NSURLConnection is deprecated and causing compilation failure (deprecation turns to error) when you bump the deployment target then I think we should silence that failure and still run it because a user may have done that and we want to make sure EarlGrey can properly synchronize with it. We use the availability macros only for cases where API is completely removed - in this case it's deprecated not removed.

Reply:
I suppressed the deprecation error for this call.